docs/governance/fees/partner-fee.md

```diff
@@ -0,0 +1,88 @@
+---
+title: Partner fee
+id: partner-fee
+---
+
+
+### Partner Fee Calculation
+
+Partner fee is calculated as the sum of all transactions traded through your project's Widget with Partner Fee defined
+in the `appData`, multiplied by the % percentage of the Partner Fee bps defined in the [`appData`](/cow-protocol/reference/core/intents/app-data).
+
+The Partner Fee per order is calculated based on the surplus token of the order (i.e. buy token for sell orders, and
+sell token for buy orders) and is then converted into WETH using an external price provider at the time of the
+settlement of the order.
+
+A percentage % of the Partner Fee shall be retained as a service fee to CoW Protocol (the "Service Fee") which is
+currently set at 15% of the total eligible Partner Fee amount.
+
```

Comment:
Maybe not on this iteration, but IMO we should add a section "Advantages of CoW Protocol's partner fee" to explain the advantages over other services that let the partner deal with many small amounts of tokens collected as fees.
This would help "justify" the service fee.

Reply:
If there are suggestions, please open them as issues.